modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-user-infra-machines-iso-ibm-z_{context}"]
+= Creating {op-system-first} machines
+
+Before you install a cluster on IBM Z infrastructure that you provision, you must install {op-system} on z/VM guest virtual machines for the cluster to use. Complete the following steps to create the machines.
+
+.Prerequisites
+
+* An FTP server running on your provisioning machine that is accessible to the machines you create.
+
+.Procedure
+
+. Log in to Linux on your provisioning machine.
+
+. Download the Red Hat Enterprise Linux CoreOS installation files from the link:https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.2/latest/[{op-system} image mirror].
```

Comment:
This link will be different for 4.3, but I'll open a PR to fix it at GA.

Reply:
Thank you!